cryptofeed/exchanges/huobi.py

```diff
@@ -14,27 +20,41 @@
 from yapic import json
 
 from cryptofeed.connection import AsyncConnection, RestEndpoint, Routes, WebsocketEndpoint
-from cryptofeed.defines import BUY, CANDLES, HUOBI, L2_BOOK, SELL, TRADES, TICKER
+from cryptofeed.defines import BUY, CANDLES, FILLS, HUOBI, L2_BOOK, MAKER, ORDER_INFO, SELL, TAKER, TRADES, TICKER, OPEN, SUBMITTING, CLOSED, LIMIT, MARKET
 from cryptofeed.feed import Feed
-from cryptofeed.types import OrderBook, Trade, Candle, Ticker
-
+from cryptofeed.types import OrderBook, Trade, Candle, Ticker, OrderInfo, Fill
 
 LOG = logging.getLogger('feedhandler')
 
 
 class Huobi(Feed):
     id = HUOBI
-    websocket_endpoints = [WebsocketEndpoint('wss://api.huobi.pro/ws')]
-    rest_endpoints = [RestEndpoint('https://api.huobi.pro', routes=Routes('/v1/common/symbols'))]
+    is_auth = False
+    rest_endpoints = [
```

Comment:
please undo this change - looks like the line did not change, but you used some tool that made a 1 line declaration into a 3 line.